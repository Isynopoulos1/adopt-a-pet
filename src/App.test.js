import { render, screen } from "@testing-library/react";
import App from "./App";

test("renders learn react link", () => {
  const tested = render(<App />);
  expect(tested).toMatchSnapshot();
});
