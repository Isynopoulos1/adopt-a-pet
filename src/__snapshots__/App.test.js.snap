// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav>
        <div
          class="nav-logo"
        >
          <img
            alt="Petlover"
            src="logo.svg"
          />
          <form
            class="search-form"
          >
            <input
              class="search"
              type="text"
            />
            <button
              class="search-button"
              type="submit"
            >
              ðŸ”Ž
            </button>
          </form>
        </div>
        <ul
          class="nav-links"
        >
          <li>
            <a
              aria-current="page"
              class="nav-link nav-link-active "
              href="/"
            >
              All Pets
            </a>
          </li>
        </ul>
      </nav>
      <h2>
        Loading...
      </h2>
    </div>
  </body>,
  "container": <div>
    <nav>
      <div
        class="nav-logo"
      >
        <img
          alt="Petlover"
          src="logo.svg"
        />
        <form
          class="search-form"
        >
          <input
            class="search"
            type="text"
          />
          <button
            class="search-button"
            type="submit"
          >
            ðŸ”Ž
          </button>
        </form>
      </div>
      <ul
        class="nav-links"
      >
        <li>
          <a
            aria-current="page"
            class="nav-link nav-link-active "
            href="/"
          >
            All Pets
          </a>
        </li>
      </ul>
    </nav>
    <h2>
      Loading...
    </h2>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
